releasenotes/notes/remove-deprecated-pulse-code-d98fc32cf385221a.yaml

```
@@ -0,0 +1,13 @@
+---
+upgrade:
+  - |
+    Remove from Qiskit Pulse:
```

I probably would link to the pulse module too:

```suggestion
    Remove from Qiskit Pulse (:mod:`qiskit.pulse`):
```